### ISSUE TITLE
new release?

### ISSUE THREAD

**Issue by thedavidmeister** [maidsafe/rust_sodium#115]
last release was a year ago

its impossible to publish crates to crates.io if they have a dependency on an unpublished crate

this makes changes like the recent bump to 1.4.0 (needed to unblock lots of other ecosystem crates, e.g. wasmer) impossible to use outside development until a new release is published

**Comment by dirvine** [comment]
We are looking for maintainers. Would you be interested?

**Comment by dirvine** [comment]
In the meantime, sodiumoxide is updated if that helps.

**Comment by thedavidmeister** [comment]
@dirvine yeah i've been thinking about it, and talking with my team, we seem to be forking this repo every 3-6 months for minor reasons, and are about to fork again...

i am interested in helping, although it would only be the occasional update :)
